src/main/java/hlf/java/rest/client/service/impl/NetworkStatusImpl.java

```diff
@@ -355,32 +362,54 @@ public ResponseEntity<ClientResponseModel> addOrgToChannel(
           new ClientResponseModel(ErrorConstants.NO_ERROR, ErrorCode.SUCCESS.getReason()),
           HttpStatus.OK);
     } else {
-      log.warn("Network and User cannot be NULL: " + "Network = " + network + "and User = " + user);
+      log.warn(
+          "Network, User and MSP details cannot be NULL: "
+              + "Network = "
+              + network
```

Comment:
Aren't these null? Would it help if concatenating null to log messages?

Reply:
Not all of these (network, user and msp details) would be null at the same time. The intention is if any of these are null and others are not null, the log messages will help us in knowing which value is not present, which would help us in debugging.